Interfaces/about.html

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="../css/main.css" />
+    <link rel="stylesheet" href="../css/style.css" />
+    <link rel="stylesheet" href="../css/about.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.9.1/font/bootstrap-icons.css"
+    />
+    <title>About</title>
+  </head>
+  <body>
+    <header class="header-container">
+      <div class="circle"></div>
+      <nav>
+        <ul class="first-navbar">
+          <li><i class="bi bi-facebook"></i></li>
+          <li><i class="bi bi-twitter"></i></li>
+          <li><a>English</a></li>
+          <li><a>My account</a></li>
+          <li><a>logout</a></li>
+        </ul>
+      </nav>
+      <nav class="navbar">
+        <a href="../index.html"
+          ><img
+            class="brand-logo"
+            src="../img/logo.png"
+            alt="Brand-name-and-logo"
+        /></a>
+        <ul class="nav-list-items">
+          <li class="list-item"><a href="../index.html">Home</a></li>
+          <li class="list-item">
+            <a href="../Interfaces/About.html">About</a>
+          </li>
+          <li class="list-item"><a href="#">Program</a></li>
+          <li class="list-item"><a href="#">Sponsor</a></li>
+          <li class="list-item"><a href="#">New</a></li>
+          <li class="list-item"><a href="#">Ticket</a></li>
+        </ul>
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you replace the above navigation options with the ones containing your **About** page to make it more professional and easy to be navigated through. Right now, any of them can not redirect the user to one specific section of the page. I suggest you replace them with the sections' `id`s of the **About** page such as **Past Event, JavaScript Mastery Pro as '"Detail",...**

- [ ] Kindly make sure the **About navigation bar** contains an option that allows the user to go back to the **Home** page. Right now, you have it in your navigation list but it doesn't display in the **Desktop version**. Make sure you fix it.
![Screenshot from 2022-09-22 12-30-01](https://user-images.githubusercontent.com/83042742/191724738-0f10885c-40ab-4496-b659-789bcd9a9d46.png)